awx/api/views/root.py

```diff
@@ -40,19 +40,19 @@
 class ApiRootView(APIView):
     permission_classes = (AllowAny,)
     name = _('REST API')
-    versioning_class = None
```

Comment:
In this "mode" calls to `reverse()` return full urls i.e. `https://localhost:8013/api/v2/hosts`